Libraries/LibGUI/GFilePicker.cpp

```diff
@@ -30,10 +30,10 @@ Optional<String> GFilePicker::get_open_filepath()
 
 Optional<String> GFilePicker::get_save_filepath(const String& title, const String& extension)
 {
-    GFilePicker picker(Mode::Save, String::format("%s.%s", title.characters(), extension.characters()));
+    auto picker = GFilePicker::construct(Mode::Save, String::format("%s.%s", title.characters(), extension.characters()));
```

Comment:
This is already fixed in LibGUI, please rebase on top of master. :)

Reply:
Whoops, fixed now.